lightkurve/lightcurve.py

```diff
@@ -608,6 +609,15 @@ def time(self):
         """Time measurements"""
         return self.hdu[1].data['TIME'][self.quality_mask]
 
+    @property
+    def date(self):
+        """Returns the human-readable date for all good-quality cadences."""
+        bjd = self.time + 2454833.
+        jd = bjd - self.hdu[1].data['TIMECORR'][self.quality_mask]
+        jd += (0.25 + 0.62 * (5 - self.hdu[1].header['TIMSLICE'])) / 86400.
```

Comment:
@barentsen curiosity here: does this factor of ``86400`` has anything to do with the cadence type (long, short, super-long)?